### PR TITLE
Cb 112 develop trinity module less concern for optimizing

### PR DESCRIPTION
**Pull request by tfalkarkea**
The much bedreaded Trinity module...it works! As long as you are using AWS EC2 instance. The module is straightforward, I updated the container and made some changes to allow more memory for Trinity, as well as added some new customizable parameters.

You can try to run locally, I grabbed tiny test read inputs from [here (left)](https://drive.google.com/file/d/1s3Bk0PCHhLVhvJR3yxHIZCfIS_YTlC5n/view?usp=share_link) and [here (right)](https://drive.google.com/file/d/1oJLkg1C4jq2yCSEvcgovS3vEjD7ab7kB/view?usp=share_link).

Run it with your favorite choice:
```
nextflow run trinity.nf -profile test_trinity,docker
```
or
```
nextflow run trinity.nf -profile docker --pairedreads "localdata/reads/reads.{left,right}.fq.gz"
```
The sandbox EC2 has Nextflow installed, for conveniently running this module successfully.

### CODE REVIEW

File: conf/modules.config [hunk 1]
```diff
@@ -59,6 +59,19 @@ process {
         ]
     }
 
+    withName: TRINITY {
```
Comment (lzholl) [inline conf/modules.config:62]: This chunk conflicts with lines 119-130. Had to delete that for it to run.

File: nextflow.config [hunk 2]
```diff
@@ -37,6 +37,9 @@ params {
     report_zero_counts          = true
     confidence                  = 0.5
 
+    // trinity options
+    min_kmer_cov                = 2
+    noramlize_assembly          = true
 
     // Trinity options
     min_contig_length           = 200
```
Comment (lzholl) [inline nextflow.config:43]: combine these and add `min_contig_length` to the modules.config file. I thought the default was already set to 200, but that's only for the inchworm step. The final contig length default is currently set to 100, so we need to change it still.

**Review by lzholl — CHANGES_REQUESTED** [review]
Requesting a couple changes. Once I removed the conflicting configs, it ran just fine on the sandbox (I used the minigut reads on the sandbox and it took ~17 min). The tiny test dataset couldn't complete on my local machine within the given 1 hr time limit, as we expected. I did get it to get a bit farther by adding `arm` to the profiles (it's sitting at 93% complete).

File: nextflow.config (hunk 2 above)
Comment (tfalkarkea) [reply nextflow.config:43]: Can you elaborate on which contig lengths go where? This comment isn't very clear to me.

Comment (lzholl) [reply nextflow.config:43]: Disregard. I just can't read apparently.

**Comment by tfalkarkea** [comment]
@lzholl I made the changes finally!

**Review by lzholl — APPROVED** [review]

**Pull request merged**